### DISCUSSION TITLE
player: programmatically change idle-time-limit

### DISCUSSION THREAD

**Discussion by MarkRoddy** [asciinema/discussions#158] [Ideas]
Use Case: I have a video embedded on a page in my app, and I have a button that lets user toggle between a preset idle time out and no idle timeout to give users the option to watch a truncated copy of the video or one in real time. Unfortunately, this requires re-loading the page to reset the `idle-time-limit` value on the DOM element.

**Comment by ku1ik** [comment]
Nice idea although not a very common use-case. It also brings more questions, e.g. how should the translation of current time work when switching back and forth between limit/no-limit.
